fvm/context.go

```diff
@@ -46,6 +47,8 @@ type Context struct {
 	// AllowProgramCacheWritesInScripts determines if the program cache can be written to in scripts
 	// By default, the program cache is only updated by transactions.
 	AllowProgramCacheWritesInScripts bool
+
+	debug.EVMTracer
```

Comment:
This seems a bit off, I think FVM context can have a boolean about EVM but it should not be dependent on the internal EVM packages.

Reply:
I did it that way the first time, but then I need other things beside just boolean, like I need more configuration, I need name of the bucket, I need logger and possibly some other config that's just present in the builder, so I can create all these settings as separate values on the context if that's better you think, but that adds bunch of stuff that's EVM specific that's why I didn't like it and went with the approach of just adding in case you need it. Also embedding was a pattern I noticed for other things. Let me know what is the way (first time doing such a configuration).